lint/base_linter/node_linter.py

```diff
@@ -146,11 +147,45 @@ def find_local_executable(self, start_dir, npm_name):
                     self.notify_failure()
                     raise linter.PermanentError()
 
-                # A 'package.json' not yet installed?
+                # No 'node_modules' folder ?
                 if not os.path.exists(os.path.join(path, 'node_modules', '.bin')):
```

Comment:
We shouldn't test this here bc we're looking for package.json files which declare a dependency in this loop.